tests/templates/test_embeddings/test_displacement_emb.py

```diff
@@ -20,6 +20,11 @@
 import pennylane as qml
 
 
+def test_has_matrix():
+    """Test has_matrix for DisplacementEmbedding is False."""
+    assert not qml.DisplacementEmbedding.has_matrix
```

Comment:
We were planning to define a default matrix property, using the decomposition...maybe these tests are not so important?

Relates to the above comment: do you want to test that the templates have no matrix defined, or do you want to test that your query function works? Here you are doing the former, but I'm not sure if this is something we have to test, since it is just a missing feature!